feinstaub/sensors/management/commands/export_as_csv.py

```diff
@@ -127,3 +138,27 @@ def _dates(start, end):
         while current <= end:
             yield current
             current += datetime.timedelta(days=1)
+
+    @staticmethod
+    def _upload_csv(tmp_path, dest_filename, content_type):
+        bucket_name = os.getenv('AWS_BUCKET_NAME')
+        access_key = os.getenv('AWS_ACCESS_KEY')
+        secret_access_key = os.getenv('AWS_SECRET_ACCESS_KEY')
+        url_prefix = os.getenv('AWS_URL_PREFIX')
+        region = os.getenv('AWS_REGION')
+
+        dest_path = os.path.join(url_prefix, dest_filename)
+        url = 'http://s3-%s.amazonaws.com/%s/%s' % (region, bucket_name, dest_path)
+
+        session = boto3.Session(aws_access_key_id=access_key, aws_secret_access_key=secret_access_key,
+                                region_name=region)
+        s3 = session.resource('s3')
+
+        bucket = s3.Bucket(bucket_name)
+        try:
+            bucket.upload_file(tmp_path, dest_path, ExtraArgs={'ContentType': content_type,
+                                                               'ContentDisposition': 'attachment',
+                                                               'ACL': "public-read"})
+            return url
+        except IOError as e:
+            return None
```

Comment:
How is this command used?

Also, should we try and add it to our own Django app instead to avoid conflicts in future?

cc @phillipahereza as our resident Django aficionado.

Reply:
@DavidLemayian we've setup daily cron jobs for the command i.e `python3 manage.py export_as_csv --no_exclude  --type sds011 --upload_s3 True`

Yea will replicate it to our own Django app.